code/src/App.js

```diff
@@ -1,12 +1,28 @@
-import React from 'react'
-import data from './data.json'
+import React from 'react';
+import data from './data.json';
+import { Album } from "./Album";
 
-console.log(data)
+//array of albums to fetch album data from
+const albumArr = data.albums.items;
 
-export const App = () => {
-  return (
-    <div>
-      Find me in src/app.js!
+export const App = () =>
+  (
+    <div className="master-container">
+      <h1 className="site-header">New albums &#38; singles</h1>
+      <div className="header-line"></div>
+      <div className="albums">
+        {albumArr.map((album) =>
+
+          //Create one album component for each item in the album array
+          <Album
```

Comment:
I'm surprised this worked, you should wrap this component in parentheses (as in line 35)

Reply:
Hmm ... I think the Album component does have parenthesis - in line 14 and 26. It's wrapped inside the `map` function. We omitted `{}` and `return` after `=>`  since the Album component is the only thing we're returning from the` map()`. Let me know, if I misunderstood you.

Reply:
I checked it out and your flow is proper, my bad Rebeka :)